src/status_im/contexts/keycard/factory_reset/view.cljs

```diff
@@ -0,0 +1,95 @@
+(ns status-im.contexts.keycard.factory-reset.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.common.events-helper :as events-helper]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- reset-card
+  []
+  (rf/dispatch
+   [:keycard/factory-reset
+    {:on-success (fn []
+                   (rf/dispatch [:navigate-back])
+                   (rf/dispatch [:keycard/disconnect])
+                   (rf/dispatch [:open-modal :screen/keycard.factory-reset.success]))
+     :on-failure (fn []
+                   (rf/dispatch [:navigate-back])
+                   (rf/dispatch [:keycard/disconnect])
+                   (rf/dispatch [:open-modal :screen/keycard.factory-reset.fail]))}]))
+
+(defn- connect-and-reset
+  [key-uid]
+  (rf/dispatch
+   [:keycard/connect
+    {:key-uid    key-uid
+     :on-success reset-card
+     :on-error   (fn [error]
+                   (if (or (= error :keycard/error.keycard-wrong-profile)
+                           (= error :keycard/error.keycard-blank))
+                     (do
+                       (rf/dispatch [:navigate-back])
+                       (if (= error :keycard/error.keycard-wrong-profile)
+                         (do
+                           (rf/dispatch [:keycard/disconnect])
+                           (rf/dispatch [:open-modal :screen/keycard.different-card]))
+                         (rf/dispatch [:keycard/on-application-info-error error])))
+                     (reset-card)))}]))
+
+(defn success-view
+  []
+  [:<>
+   [quo/page-nav
+    {:icon-name :i/close
+     :on-press  events-helper/navigate-back}]
+   [quo/page-top
+    {:title            (i18n/label :t/keycard-reset-success)
+     :description      :text
+     :description-text (i18n/label :t/keycard-empty-ready)}]
+   [rn/view {:style {:flex 1}}]
```

Comment:
Quick one, what is this `rn/view` for? It doesn't have any content or size, that's why I'm asking. File `src/status_im/contexts/keycard/different_card/view.cljs` also has this same empty `rn/view`.

Reply:
@flexsurfer probably my comment got lost in the myriad of comments in the PR, but could you elaborate on my question above?

Reply:
oh sorry @ilmotta  that true, I missed your comment, this view just pushes next view to the bottom

Reply:
Ah okay it's really necessary. Thanks for the explanation!